ci/build_cpp.sh

```diff
@@ -9,12 +9,15 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+LIBRMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1404 cpp)
+
```

Comment:
We will revert changes to CI scripts that use upstream artifacts before merging.